inference-engine/src/multi_device/multi_device_plugin.cpp

```diff
@@ -205,6 +222,14 @@ ExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadExeNetworkImpl(co
                                                           enablePerfCounters);
 }
 
+// Is called only when caching is enabled
+InferenceEngine::ExecutableNetwork MultiDeviceInferencePlugin::LoadNetwork(const std::string& modelPath,
+                                                                           const std::map<std::string, std::string>& config) {
+    CNNNetwork network;
+    auto res = DoLoadNetwork(modelPath, network, config);
+    return GetCore()->ToExecutableNetwork(res, GetMetric(METRIC_KEY(FULL_DEVICE_NAME), {}));
```

Comment:
`FULL_DEVICE_NAME` provides information in form like:
```
Intel(R) Core(TM) i9-10920X CPU @ 3.50GHz
Intel Movidius Myriad X VPU
```
And this is not what we except. Maybe just `GetName()` ?

Reply:
I'm actually calling this for 'Multi' plugin, just didn't want to hardcode 'MULTI'